### PR TITLE
Update install.rst to match the newest code

### PR DESCRIPTION
**Pull request by NewUserHa**
enhancing the Excel files related documents of pandas

### CODE REVIEW

**Comment by mroeschke** [comment]
Thanks for the PR but I think the order of these dependencies is fine as is so closing

**Comment by NewUserHa** [comment]
the important changes are completing the usage of notes for each depencies for excel, and adding the missing 'odfpy' which exists in pyproject.toml but not in documents.

the order of these dependencies is not important

**Comment by mroeschke** [comment]
OK sorry the diff did not make that obvious. Could you remove the re-ordering so the diff is clearer?

**Comment by mroeschke** [comment]
The ordering in either is not important so I would keep the existing ordering for clarity

**Comment by NewUserHa** [comment]
ok the re-ordering is removed

File: doc/source/getting_started/install.rst [hunk 1]
```diff
@@ -277,11 +277,12 @@ Installable with ``pip install "pandas[excel]"``.
 ========================= ================== =============== =============================================================
 Dependency                Minimum Version    pip extra       Notes
 ========================= ================== =============== =============================================================
-xlrd                      2.0.1              excel           Reading Excel
-xlsxwriter                3.0.5              excel           Writing Excel
-openpyxl                  3.1.0              excel           Reading / writing for xlsx files
+xlrd                      2.0.1              excel           Reading / writing for xls files
```
Comment (mroeschke) [inline doc/source/getting_started/install.rst:280]: xlrd doesn't write in pandas

Comment (NewUserHa) [reply doc/source/getting_started/install.rst:280]: but xlrd the dependency itself can write.
How should write note for this?

Comment (mroeschke) [reply doc/source/getting_started/install.rst:280]: `Reading xls files`

Comment (NewUserHa) [reply doc/source/getting_started/install.rst]: ok

**Review by mroeschke — APPROVED** [review]

**Comment by mroeschke** [comment]
Thanks @NewUserHa

**Pull request merged**